testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/util/UIUtils.java

```diff
@@ -74,6 +74,13 @@ public static void clickLink(WebElement element) {
 
         waitUntilElement(element).is().clickable();
 
+        performOperationWithPageReload(BrowserDriverUtil.isDriverChrome(driver)
+                ? () -> element.sendKeys(Keys.ENTER)
+                : element::click);
+    }
+
+    public static void click(WebElement element) {
```

Comment:
Is it possible to add javadoc for this method (EG. mention that is useful for selects etc)? It can be good to do the same for `clickLink` and mention when this one might be useful.

Reply:
Yep, no problem if @miquelsi and @pedroigor agree this is the way to go I will add the javadoc.

Reply:
javadocs added for the two methods.